schema/Review.schema.yaml

```diff
@@ -0,0 +1,30 @@
+title: Review
+'@id': schema:Review
+extends: CreativeWork
+role: primary
+status: stable
+category: metadata
+description: A review of an item, e.g of an Article, or SoftwareSourceCode.
+$comment: |
+  Use the `content` property for the structured content of the review in
+  preference to the schema.org `text` or `reviewBody` properties which
+  have plain text as the expected type of this property.
+properties:
+  itemReviewed:
+    '@id': schema:itemReviewed
+    description: The item that is being reviewed.
+    $ref: Thing
+  reviewAspect:
+    '@id': schema:reviewAspect
+    description: The part or facet of the item that is being reviewed.
+    anyOf:
+      - type: integer
+      - type: string
+  comments:
+    '@id': stencila:comments
+    description: Comments, usually on specific parts of the item reviewed, that form part of the review.
+    $comment: |
+      This is a custom property which allows us to associate Comments with a review.
```

Comment:
Is this the same as the [`schema:comment` property](https://schema.org/comment)?

Reply:
Yes, it is. Good spot, thanks! I hadn't noticed `schema:comment`, I should have checked.